java/ddlogapi/DDlogCommand.java

```diff
@@ -27,11 +27,11 @@
  * </p>
  */
 public interface DDlogCommand<R> {
-    public enum Kind {
+    enum Kind {
```

Comment:
Why shouldn't it be public?

Reply:
Intellij told me that it's redundant; I think that if it is in an interface it is always public.

Reply:
But it is indeed clearer if you mark it public...